### DISCUSSION TITLE
SFCC -SFMC connector

### DISCUSSION THREAD

**Discussion by mkumar8687** [SalesforceCommerceCloud/sfcc-ci#434] [Team Posts]
Hello all,
I want to implement SFCC-SFMC connector for Storefront. If any of You any doc related to "SFCC-SFMC" Connector plz do share with me. Thanks in Advance.

**Comment by pchug3** [comment]
https://github.com/SalesforceCommerceCloud/marketing-cloud-connector/tree/V2.0.1 Please refer the link for the integration

**Comment by vaideeshn** [comment]
Lot of documentation available on exchange - https://xchange.demandware.com/docs/DOC-43523
https://xchange.demandware.com/search.jspa?q=Salesforce%20Marketing%20cloud

**Comment by mkumar8687** [comment]
Thank you @vaideeshn and @pchug3.

**Comment by rohanmallick08** [comment]
You can refer the SFMC connector Github page. It has the implementation
details with the connector cartridge.

https://github.com/SalesforceCommerceCloud/marketing-cloud-connector

**Comment by navjotbhandari** [comment]
Is there any Marketing Cloud sandbox/demo account to look for?
Thanks in advance

**Comment by nimitshukla123** [comment]
Hi

 Marketing Cloud do not provide a developer edition sandbox.

Thanks

On Wed, Jul 17, 2019 at 4:21 AM navjotbhandari <notifications@github.com>
wrote:

> Is there any Marketing Cloud sandbox/demo account to look for?
> Thanks in advance
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/SalesforceCommerceCloud/teams/community/discussions/85/comments/5?email_source=notifications&email_token=ADMLMX2BNKOV3UXQFJJ73BDP7YOXDA5CNFSM4IEDFQSKYY3PNVWWK3TUL52HS4DFWNCGS43DOVZXG2LPNZIG643UKJSXA3DZVJRW63LNMVXHIX3JMTHAAAQKZE>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADMLMX56DY2JVN4CRBJWBTLP7YOXDANCNFSM4IEDFQSA>
> .
>


-- 

Regards



*Nimit Shukla*

T: +65 84575208

**Comment by navjotbhandari** [comment]
Thanks for response..
Does anyone has rough estimate of efforts required to implement SFMC (just transactional emails functionality) not the abandon cart.
Thanks
Nav

**Comment by intel352** [comment]
@navjotbhandari from what I recall it could be a few days up to 1-2 weeks depending on your familiarity with SFMC, and whether you run into issues requiring troubleshooting. The setup isn't very technical, just more time consuming.

**Comment by navjotbhandari** [comment]
yea..thats make sense .. thanks for info :)

**Comment by navjotbhandari** [comment]
@intel352 looks like Client want to start with marketing cloud to service cloud connector first.
any pointers to documentation on implementing that part. 
thanks in advance

**Comment by philipegan** [comment]
@navjotbhandari are you referring to Marketing Cloud Connect, which allows customers to connect Marketing Cloud in to Sales/Service Cloud?  If so, here's a pointer to some info: https://help.salesforce.com/articleView?id=mc_co_marketing_cloud_connect.htm&type=5
